subjects/multorsum/README.md

```diff
@@ -0,0 +1,35 @@
+## multorsum
+
+### Instructions
+
+Write a function that receives a slice of int and an int representing the initial value. If the int is odd it should multiply the accumulated value. If it is even, it should be added instead. Return the result. If the slice is empty return 0.
```

Comment:
Is this the same as you said? Im a bit confused with the readme...

Write a function that receives two arguments, a slice of int and an int representing the initial value.

- If the initial value is odd it should multiply the accumulated value of the slice.
- If it is even, it should be added instead.
- If the slice is empty return 0.

Reply:
Thanks for pointing this out, I'll be more specific regarding the initial value. The numbers that define whether is going to be multiplied or added are the ones from the slice.